src/target/stm32f1.c

```diff
@@ -254,6 +258,87 @@ bool at32fxx_probe(target_s *t)
 	return false;
 }
 
+/* Identify MM32 devices (Cortex-M0) */
+bool mm32l0xx_probe(target_s *t)
+{
+	uint32_t mm32_id;
+	const char *name = "?";
+	size_t flash_kbyte = 0;
+	size_t ram_kbyte = 0;
+	size_t block_size = 0x400;
+
+	if (!t)
+		return false;
+	if (target_mem_read(t, &mm32_id, DBGMCU_IDCODE_MM32L0, sizeof(mm32_id))) {
+		DEBUG_WARN("mm32xx_probe: read error at 0x%" PRIx32 "\n", (uint32_t)DBGMCU_IDCODE_MM32L0);
+		return false;
+	}
+	switch (mm32_id) {
+	case 0xcc568091U:
+		name = "MM32L07x";
+		flash_kbyte = 128;
+		ram_kbyte = 8;
+		break;
+	case 0xcc56a097U:
+		name = "MM32SPIN27";
+		flash_kbyte = 128;
+		ram_kbyte = 12;
+		break;
+	case 0x0:
```

Comment:
Please write the constant out in full for consistency and apply "U" markings to both this and the next (especially the next due to sign extension problems!)